rundeckapp/grails-app/services/rundeck/services/NodeStatusService.groovy

```diff
@@ -118,28 +147,55 @@ class NodeStatusService implements RundeckProjectConfigurable, InitializingBean
         result
     }
 
-    CacheNodeStatus getNodeStatus(String project, INodeEntry node, String user, String userRolelist){
+    void mergeNodeAttributes(String project, INodeSet nodeSet){
+        //merged attributes with health check cache
+        Map<String, CacheNodeStatus> cacheStatus = getCurrentStatus(project)
+        for (final INodeEntry iNodeEntry : nodeSet.getNodes()) {
+            if(cacheStatus.get(iNodeEntry.nodename)) {
+                def status =  cacheStatus.get(iNodeEntry.nodename)
+                iNodeEntry.attributes?.put("checkExecutor", status.executorReachable)
```

Comment:
i wish `checkExecutor` was a configurable attribute, or maybe we can just agree on some other name, like "healthCheckStatus" @ahonor ?

Reply:
is this OK @gschueler ?

<img width="711" alt="screenshot 2018-12-24 13 46 24" src="https://user-images.githubusercontent.com/1669317/50405984-75d92080-079b-11e9-8fad-b85654d82492.png">